internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/mattermost/mattermost-cloud/model"
 )
 
+// DefaultKubernetesVersion is the default value for a kubernetes cluster
+// version value.
+const DefaultKubernetesVersion = "0.0.0"
```

Comment:
what means 0.0.0 which version will be that? sounds confusing to me

Reply:
Whenever you see `0.0.0` here, it is just the version we store in the database for the kubernetes version until we know what it really is. For example, a newly created cluster will have this value until it is created, then the database will be updated to the actual running version.

Reply:
thanks!